ansible/configs/sap-integration/post_software.yml

```diff
@@ -0,0 +1,371 @@
+      delay: 60
+
+    - name: Cluster-info
+      command: oc cluster-info
+      register: clusterinfor
+      changed_when: false
+
+    - name: Create project
+      command: oc new-project postflightcheck
+      register: newproject
+
+    - name: New-app
+      command: oc new-app cakephp-mysql-persistent -n postflightcheck
+      register: newapp
+
+    - name: Wait for mysql
+      command: timeout 300 oc rollout status dc/mysql -w -n postflightcheck
+      register: mysqlw
+      changed_when: false
+
+    - name: Wait for php
+      command: timeout 300 oc rollout status dc/cakephp-mysql-persistent -w -n postflightcheck
+      register: phpw
+      changed_when: false
+      retries: 2
+      delay: 60
+      until: phpw is succeeded
+
+    - name: Get route
+      command: >-
+        oc get route
+        -l template=cakephp-mysql-persistent
+        --no-headers
+        -o json
+        -n postflightcheck
+      register: getroute
+      changed_when: false
+      retries: 10
+      delay: 5
+      until: getroute is succeeded
+
+    - name: Test that route is reachable
+      uri:
+        url: "http://{{ getroute.stdout|from_json|json_query('items[0].spec.host') }}"
+      register: testroute
+      retries: 15
+      delay: 5
+      until: testroute is succeeded
+
+    - name: Delete project
+      command: oc delete project postflightcheck
+
+    - name: Switch back to default project
+      command: oc project default
+
+    - debug:
+        msg: "{{ item }}"
+      loop:
+      - "user.info: "
+      - "user.info: Post Flight Check"
+      - "user.info: DNS Web Console ............... {{ 'OK' if checkdnswebconsole.rc == 0 else 'FAIL' }}"
+      - "user.info: DNS API ....................... {{ 'OK' if checkdnsapi.rc == 0 else 'FAIL' }}"
+      - "user.info: Web console ................... {{ 'OK' if testwebconsole is succeeded else 'FAIL' }}"
+      - "user.info: API ........................... {{ 'OK' if clusterinfor.rc == 0 else 'FAIL' }}"
+      - "user.info: Create Project with PV ........ {{ 'OK' if newproject.rc == 0 else 'FAIL' }}"
+      - "user.info: App deployed .................. {{ 'OK' if phpw.rc == 0 and mysqlw.rc == 0 else 'FAIL' }}"
+      - "user.info: Route ......................... {{ 'OK' if testroute is succeeded else 'FAIL' }}"
+
+    - name: Fail smoke tests
+      when:
+      - run_smoke_tests | default( False ) | bool
+      - >-
+        checkdnswebconsole.rc != 0
+        or checkdnsapi.rc != 0
+        or testwebconsole is failed
+        or clusterinfor.rc != 0
+        or newproject.rc != 0
+        or phpw.rc != 0
+        or mysqlw.rc != 0
+        or testroute is failed
+      fail:
+        msg: "FAIL Smoke tests"
+      ignore_errors: no
+
+# Deploy Workloads
+- name: Step 005.2 - Deploy Infra and Student Workloads
+  import_playbook: workloads.yml
+
+# Workarounds to enable cluster shutdown before 24h are over
+- name: Step 005.3 - Enable Cluster Shutdown and Resume
```

Comment:
You're using a version of OpenShift that supports this without a workaround.

Reply:
done